### ISSUE TITLE
cassandra-tracing fail to extract traces from Scylla

### ISSUE THREAD

**Issue by tzach** [Knewton/cassandra-toolbox#4]
Tested cassandra-tracing with scylla 1.3 and got an error. Full details below

running scylla
```
sudo docker run --name some-scylla -d scylladb/scylla --smp 1 --memory 1G
```

Generating traces

```
sudo docker exec -it some-scylla cqlsh
cqlsh> tracing on;                           
Now Tracing is enabled
cqlsh> select * from system.peers;  
```

Using cassandra-tracing 
```
$ cassandra-tracing -t 1 172.17.0.2 
100% Complete: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX|100


Total skipped due to null duration:	0
	
Total skipped due to error:	8
Traceback (most recent call last):
  File "/usr/bin/cassandra-tracing", line 322, in <module>
    main()
  File "/usr/bin/cassandra-tracing", line 316, in main
    args.tombstone_threshold
  File "/usr/bin/cassandra-tracing", line 182, in process_sessions
    print('\t' + '\n\t'.join(skipped_err[0] + '\t' + skipped_err[1]))
TypeError: can only concatenate tuple (not "str") to tuple
```

**Comment by wickmaster** [comment]
@tzach: Thanks for alerting us to this issue.  Both facets — the bug leading to the traceback and the difference leading to Scylla traces being skipped due to error — have been fixed in version 0.1.5.  Please see pull request #5 for details.

**Comment by tzach** [comment]
Thanks @wickmaster 
Looks good.
For completeness, here is the output with the latest version of cassandra-tracing, and the same setup as https://github.com/Knewton/cassandra-toolbox/issues/4 , where 172.17.0.4 is Scylla IP

```
$ cassandra-tracing -t 1 172.17.0.4
100% Complete: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX|100


Total skipped due to null duration:	0
	
Total skipped due to error:	0



1 sessions satisfying criteria.
               Session Id                Duration (microsecs)    Max Tombstones     Total Tombstones    Flags            Time Started         Query
  3b8d2e60-82f7-11e6-80ee-000000000001           3062                  0                   0                      2016-09-25T08:08:24.902000  select * from system.peers;
EOM
```
